### PR TITLE
[PROPOSAL] Revising documentation on the Developer Site *DO NOT MERGE*

### PR DESCRIPTION
**Pull request by ALRubinger**
* And potentially couple this with Developer Site content refresh
* This is ALL up for debate and alignment - in particular we have to work through this first draft until it accounts for all perspectives and we are aligned on goals enough to split up work

### CODE REVIEW

File: proposals/open-source-programs/PROPOSAL_001.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+## Implementation Considerations
+* Embed into every Project section a link to contribute - and make sure it's offshoring into appropriate homes
+* Link out to FAQs and Glossary on the Dev Site where appropriate (and/or tooltip)
+* Embed feedback forms where appropriate
+* Pop outs to choose software version where tutorials are, or where code is embedded
+* Embed media into guides where appropriate, perhaps cross-linking into where they're "housed" in Media section. Perhaps the "embed" is exandable as an option and doesn't take up lots of real estate unless user opts into.
+
+## External Examples
+* [Use Case Approach from Mattr](https://learn.mattr.global/docs/)
+* From @dcrusso: [leverage GitHub Pages to do really simple playground](https://github.com/TBD54566975/web5-components/blob/main/.github/workflows/Pages.yml) NOTE: @bobbilee19 suggests calling this "sandbox". Align on language?
```
Comment (dcrousso) [inline proposals/open-source-programs/PROPOSAL_001.md:180]: s/dcrusso/dcrousso :(

Comment (ALRubinger) [reply proposals/open-source-programs/PROPOSAL_001.md]: Fixed in 8933d150c3bfdb4fca4017b0fb8148399fa4b9b4 with my apologies @dcrousso!

**Comment by ALRubinger** [comment]
@bobbilee19: Ready to walk through this! There are things I know needs changing here; I'm holding on those until we link up and get your feedback incorporated and this gets cleaned up.

**Comment by ALRubinger** [comment]
Addressed by re-coordinated Phase 2 and 3 work

**Pull request closed without merging**